docs/interactions/web-messaging/webview.md

```diff
@@ -0,0 +1,21 @@
+# Webview
+
+The webview is a convinient way of opening a custom made webpage. It can be chosen by setting the optional target field to `webview` when sending a Structured Message containing a link element, such as [Rich Link](../structured-message/rich-link), [Carousel](../structured-message/carousel), or [Template](../structured-message/template).
```

Comment:
```suggestion
The webview is a convinient way of opening a custom made webpage. It can be chosen by setting the optional target field to `webview` when sending a Structured Message containing a link element, such as [Rich Link](../../structured-messages/rich-link), [Carousel](../../structured-messages/carousel), or [Template](../../structured-messages/template).
```
Links are broken in local mkdocs